src/components/Summary/Summary.jsx

```diff
@@ -27,14 +28,32 @@ function Summary({
   const [summaryError, setSummaryError] = useState(null);
 
   useEffect(() => {
-    generateResponse(articleSummaryData.mainContent)
-      .then((summary) => {
-        setSummaryText(summary);
-      })
-      .catch(() => {
-        setSummaryError("요약 생성 중 에러가 발생했습니다. 다시 시도해주세요.");
-      });
-  }, [articleSummaryData.mainContent]);
+    const summaryDatas =
+      JSON.parse(window.sessionStorage.getItem("summarys")) || {};
+    const articleId = articleSummaryData?.id;
+
+    if (summaryDatas?.[articleId]) {
```

Comment:
```suggestion
    if (summaryDatas.[articleId]) {
```

`const summaryDatas = JSON.parse(window.sessionStorage.getItem("summarys")) || {};`
31번째 라인에서 sessionStorage에 summarys 값이 없는 경우 summaryDatas에 빈 객체를 할당하고 있으므로 옵셔널 체이닝을 사용하지 않아도 될 것 같습니다.